lib/teleterm/services/connectmycomputer/connectmycomputer.go

```diff
@@ -358,6 +360,14 @@ func (n *NodeJoinWait) Run(ctx context.Context, accessAndIdentity AccessAndIdent
 		return clusters.Server{}, trace.Wrap(err)
 	}
 
+	// By default, the node config includes a command label with a hostname. Immediately after joining
+	// the cluster, the label is most likely not going to be updated. To work around that, we fill it
+	// out with os.Hostname if it's empty.
```

Comment:
What this comment is pointing out isn't quite clear to me (granted it could just be a function of the fact that I'm unfamiliar with this part of the codebase).

Reply:
Oh, I think the comment doesn't make it clear that the config comes from `teleport node config` that we call during the setup of Connect My Computer.

I'll update the comment to my best ability and then merge this PR, but please let me know if the updated comment is any better.